apps/sms/views/conversation/js/conversation.js

```diff
@@ -440,6 +440,8 @@ var ConversationView = {
   },
 
   getIdIterator: function conv_getIdIterator() {
+    // TODO: Will need to replace with Conversation service method or caching
+    // the ids locally.
     return Threads.active.messages.keys();
```

Comment:
I think we can use it now; we'll especially need the service method when applying the changes from the select form, but not necessarily here. Maybe we'll need to be more asynchronous as well (or maybe not, I don't really know yet :D).

One possibility is having a "Thread" object in this.activeThread; this thread could have all message "ids" already, but not the messages' content, that could be fetched lazily.

So in the end, maybe we'll have `this.activeThread.messageIds()` here later.

All that to say... we don't really know how this will be later, so better use this.activeThread here now to simplify later things :)